form/without_class.rst

```diff
@@ -105,4 +105,4 @@ but here's a short example:
 
 .. code-block:: php
 
-    new NotBlank(array('groups' => array('create', 'update'))
+    new NotBlank(array('groups' => array('create', 'update')))
```

Comment:
Shouldn't the code block be part of the tip? I suggest to indent everything starting with line 106 by additional four spaces.